content/ar/docs/setup/best-practices/certificates.md

```diff
@@ -0,0 +1,285 @@
+---
+title: شهادات البنية التحتية للمفتاح العام PKI و المتطلبات
+reviewers:
+- sig-docs-ar-reviews
+content_type: المفهـــوم
```

Comment:
do not translate this one refer to ko to check more what should not be translate https://github.com/kubernetes/website/blob/main/content/ko/docs/setup/best-practices/certificates.md?plain=1